test/core/tsi/alts/handshaker/alts_tsi_handshaker_test.cc

```diff
@@ -445,7 +445,7 @@ static void on_server_next_success_cb(tsi_result status, void* user_data,
   // Validate peer identity.
   tsi_peer peer;
   ASSERT_EQ(tsi_handshaker_result_extract_peer(result, &peer), TSI_OK);
-  ASSERT_EQ(peer.property_count, kTsiAltsNumOfPeerProperties);
+  ASSERT_GE(peer.property_count, kTsiAltsMinNumOfPeerProperties);
```

Comment:
We should still know the exact number this should be given the test case, right? Either `== kTsiAltsMinNumOfPeerProperties + 1` or `== kTsiAltsMinNumOfPeerProperties`?

Reply:
+1.